koku/api/report/aws/openshift/query_handler.py

```diff
@@ -122,11 +123,15 @@ def __init__(self, parameters):
             parameters    (QueryParameters): parameter object for query
 
         """
-        self._mapper = OCPAWSProviderMap(
-            provider=self.provider,
-            report_type=parameters.report_type,
-            cost_type=parameters.cost_type,
-        )
+        kwargs = {
+            "provider": self.provider,
+            "report_type": parameters.report_type,
+            "cost_type": parameters.cost_type,
+        }
+        if markup_cost := AWS_MARKUP_COST.get(parameters.cost_type):
+            kwargs["markup_cost"] = markup_cost
+
+        self._mapper = OCPAWSProviderMap(**kwargs)
```

Comment:
These changes are totally fine with me, but we need to ensure we update the other `_mapper` instantiations too for the sake of consistency. That definitely doesn't need to happen in this PR.